src/System.Private.CoreLib/shared/System/Type.Helpers.cs

```diff
@@ -342,7 +342,7 @@ public virtual bool IsAssignableFrom(Type c)
             // For backward-compatibility, we need to special case for the types
             // whose UnderlyingSystemType are runtime implemented. 
             Type toType = this.UnderlyingSystemType;
-            if (toType.IsRuntimeImplemented())
+            if (toType?.IsRuntimeImplemented() == true)
```

Comment:
I have replied to your CoreFX comment (https://github.com/dotnet/corefx/pull/32601#discussion_r222492879) that your original version looks better to me than `== true`.

Either way is fine with me though.